### PR TITLE
Fix #1705: Missing Country Code Picker with backend functionality in …

### LINKED ISSUE #1705: Missing Country Code Picker with backend functionality in Register Activity
**Summary:**

Missing Country Code Picker with backend functionality in Register Activity

**Steps to reproduce:**

Open create new account activity

**Expected behaviour:**

Phone number field should have a country code picker.

**Observed behaviour:**

Phone number field donot have a country code picker.

**Device and Android version:**

Any

**Screenshots:**

### PR DESCRIPTION
**Pull request by ritish099**
…Register Activity

Fixes #1705

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by ritish099** [comment]
@miPlodder kindly check it out...

File: app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt [hunk 1]
```diff
@@ -4,6 +4,7 @@ import android.graphics.PorterDuff
 import android.os.Bundle
 import android.text.Editable
 import android.text.TextWatcher
+import android.util.Log
```
Comment (PatelVatsalB21) [inline app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt]: @ritish099 I think this is unused import. Have you verified with './gradlew check' ?

Comment (ritish099) [reply app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt]: @PatelVatsalB21 thanks for pointing out..

**Review by PatelVatsalB21 — APPROVED** [review]
LGTM

**Pull request closed without merging**